docs/product/tracing/index.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Tracing
+sidebar_order: 20
+description: "Learn how instrumenting tracing for both your frontend and backend can help identify and debug application performance issues. Distributed tracing will allow you to see the impact of errors across your full stack and microservices."
+---
+
+## Tracing
+
+Tracing is the process of capturing the timing and flow of requests and operations as they happen throughout your application. It can be a powerful debugging tool, helping you identify which link in a sequence of events is causing a problem.
+
+![An image of a trace with spans.](./img/The-anatomy-of-a-trace.png)
+
+### What's Distributed Tracing?
+
+Distributed tracing provides a unified view of how a single request moves from the frontend to the backend and beyond. This is particularly useful in modern applications, which are often composed of multiple services working together. To get a distributed view of your application, instrument Sentry in both your frontend and your backend.
+
+With distributed tracing, you can identify performance bottlenecks that are affecting your application no matter where in your application's workflow they occur. You can then use Sentry tools such as the [Trace Explorer](/product/explore/traces/) or [Trace View](/concepts/key-terms/tracing/trace-view/) page to dig deeper and find the root cause of the problem. Distributed tracing is particularly helpful for identifying problems you can't just debug using console logs, such as race conditions, [N+1](/product/issues/issue-details/performance-issues/n-one-queries/), and [caching issues](/product/insights/caches/).
+
+### What's a Trace?
+
+Each trace has a string of globally unique characters called a trace ID, that's shared by both frontend and backend services they help connect all the actions that take place to make an event such as a button click or a page scroll. Each trace is composed of multiple spans.
+
+### What's a Span?
+A span is a named, timed operation that represents a part of the application workflow. It provides granular insights into specific tasks, like API requests or database queries. Multiple spans are pieced together to create a trace. providing a comprehensive view of its path, timing, and interactions for troubleshooting and performance analysis.
+
+### What's a Transaction?
+
+Transactions are a Sentry-specific unit of measurement that references any event that users send to Sentry. In addition to standard event fields, transactions also contains child spans and have a unique transaction ID.
+
+## How to Use Tracing in Sentry
+
+To get the most out of tracing in Sentry, you'll need to instrument both the frontend and backend of your application with Sentry's SDK. Once this is done, there are several places where you can view and interact with tracing data:
```

Comment:
We're currently moving towards emphasizing the usefulness of partial traces as well. In other words, even if it's _only_ your frontend or backend that's instrumented and not the entire distributed system, the partial trace still provides useful context for debugging issues _within_ that app or service. I think you can still say that the best experience is by instrumenting both the frontend and the backend together, but maybe make it sound like less of a requirement in case a customer wants to get started by instrumenting only one part of their stack.